en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-connector-example.md

```diff
@@ -112,17 +108,19 @@ Follow these steps to set up the ESB Solution Project and the Connector Exporter
     
             - **queryString** :  This variable contains specified SOQL query. In this sample this SOQL query executes to retrive `id` and `name` from created `Account`. If the query results are too large, the response contains the first batch of results.
 
-        2. Navigate into the **Palette** pane and select the graphical operations icons listed under **Salesforce Connector** section. Then drag and drop the `query` operations into the Design pane.      
+        2. Navigate into the **Palette** pane and select the graphical operations icons listed under **Salesforce Connector** section. Then drag and drop the `query` operations into the Design panel.      
     
             <img src="../../../../assets/img/connectors/salesforce-drag-and-drop-query.png" title="Add query operation to retrive sequnce" width="800" alt="Add query operation to retrive sequnce"/> 
     
-        3. Select the query operation and add `id, name from Account` query to the properties section shown as bellow.
+        3. Select the query operation and add SOQL query to the properties section shown as bellow.
+           
+            - **SOQL query** : id, name from Account (Retrive list of `id`s and `name`s from the `Account` type sObject)
```

Comment:
```suggestion
            - **SOQL query** : `id` and `name` from `Account` (Retrieve list of `id`s and `name`s from the `Account` type sObject)
```

Check spelling of "retrieve" everywhere.

Reply:
Updated.